addons/im_livechat/static/src/legacy/widgets/livechat_button.js

```diff
@@ -102,7 +101,7 @@ const LivechatButton = Widget.extend({
      */
     _addMessage(data, options) {
         options = Object.assign({}, this.options, options, {
-            serverURL: this._serverURL,
+            serverURL: this.messaging.livechatButtonView.serverURL,
```

Comment:
You can probably use `this.messaging.livechatButtonView.serverURL` in `WebsiteLivechatMessage` as well, and thus remove this from the options.

Reply:
Ok, will do if it's safe.

Same reasoning as I said in another PR why I didn't do: I just wanted to simply move `LivechatButton` attributes to `LivechatButtonView`. The other fields/props/attributes look related, but I might be surprised of some shenanigans/hack/etc that would be problematic.

I tend to not trust the code, and for good reasons, because there are already too many hacks I see right now 😅

Reply:
Hm... In the end, will not convert it, just because I'm not 100% confident, and globally can be a waste of time to convert it and have red runbot. I prefer not taking this risk.

Eventually this `WebsiteLivechatMessage/serverUrl` will be introduced, and then some post-cleaning in models for related, like this one look like at a glance. This is much safer to do step by step, and globally it's a gain of time because while more changes and more diffs, all of them are very safe.